### DISCUSSION TITLE
2 PowerStreams

### DISCUSSION THREAD

**Discussion by bragg2012** [svenerbe/ecoflow_dynamic_power_adjustment#22] [Q&A]
What’s the best approach for using 2x PowerStreams (with 2 batteries or 2 PS connected to 1 battery (D2Max DPro).

**Comment by Just-Zuul** [comment]
2 Automatisations - one for each Powerstream (different S/N) one pyscript connected with the Dev-Portal.

Its easy  - the Script (pyscript) send only the data to the Dev-Portal - all other you make with the autonatisation and / or other scripts
